go/enclave/container/enclave_container.go

```diff
@@ -23,11 +23,35 @@ type EnclaveContainer struct {
 	Logger    gethlog.Logger
 }
 
-// NewEnclaveContainer wires up the components of the Enclave and its RPC server. Manages their lifecycle/monitors their status
-func NewEnclaveContainer(config config.EnclaveConfig) *EnclaveContainer {
+func (e *EnclaveContainer) Start() error {
```

Comment:
This was just moved up from below, I put the container methods with the struct and constructors at the bottom.